opencensus/proto/trace/BUILD.bazel

```diff
@@ -16,7 +16,10 @@ package(default_visibility = ["//visibility:public"])
 
 proto_library(
     name = "trace_proto",
```

Comment:
We should have 2 different proto_libraries here.

Reply:
Got it, done.